content/posts/pandoc/pandoc.md

```diff
@@ -0,0 +1,334 @@
+
+**Code explaination**
+
+- The first line adds a section tittle. This is not part of the slide, but can be used to generate table of contents, and in slide navigation.
+- The second line creates a new slide with the slide tittle.
+- To create a `itemize` as LaTeX, we can you the same method in markdown.
+- Also we can use the same method in markdow to create a figure, table inside a slide.
+
+**Result** 
+
+![](/posts/pandoc/images/result1.png)
+
+**Code comparation with LaTeX**
+
+```latex
+\documentclass[t]{beamer}  
+\setbeamertemplate{navigation symbols}{}
+\begin{document}
+\section{Section heading 1 slide}
+\begin{frame}
+\frametitle{Slide 1 of Section 1}
+\begin{itemize}
+  \item {
+   	This is a list 
+  }
+  \item {
+  	Another list item
+  }
+  \item{
+  	etc,...
+  }
+\end{itemize}
+\begin{figure}
+  \includegraphics{testImage.png}
+\end{figure}
+\end{frame}
+\section{Section 2}
+\end{document}
+```
+
+&rarr; Markdown is muchhhhhhhhh shorter line of code comparing to LaTeX with the same content.
+## Advanced configuration
+
+#### YAML
+You might want a slide contain a Beamer's theme, author name, presentation name,... - a fully features as LaTeX beamer but written in markdown. To deal with that, we can add some YAML configuration at the beginning of the markdown file.\
+Fill in all the necessary fields:
+
+- title: the presentation's name.
+- author: your name.
+- date
+- institute
+- theme: Beamer theme, you can chose one at [this site](https://hartwork.org/beamer-theme-matrix/)
+- colortheme: theme colorscheme
+- mainfont
+- toc: table of content slide ( use `true` to enable this slide)
+
+**YAML example**
+```YAML
+---
+title: "Pandoc Tutorial"
+author: Do Duy Huy Hoang 
+date: \today{}
+institute: 
+- The Code Cousins
+theme:
+- Ilmenau
+colortheme:
+- beaver 
+mainfont: "SourceSansPro-Regular"
+toc:
+- true
+---
+```
+
+**Result**
+
+    🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉🎉
+![](/posts/pandoc/images/yamlResult1.png) ![](/posts/pandoc/images/yamlResult2.png)
+
+#### Tricks and Tips
+
+**Image Scalling**
+
+The width and height attributes on images are treated specially. When used without a unit, the unit is assumed to be pixels. However, any of the following unit identifiers can be used: px, cm, mm, in, inch and %. There must not be any spaces between the number and the unit. For example:
+
+```markdown
+![](file.jpg){ width=50% }
+```
+Equal to LaTeX
+
+```latex
+\includegraphics[width=0.5\textwidth,height=\textheight]{file.jpg}
+```
+
+When no width or height attributes are specified, the fallback is to look at the image resolution and the dpi metadata embedded in the image file.
+
+**Slides with Columns in Pandoc**
+Current versions of pandoc (i.e., pandoc 2.0 and later) supports [fenced divs](https://pandoc.org/MANUAL.html#fenced-divs). Specially named divs are transformed into columns when targeting a slides format:
```

Comment:
- Current *version* ...
- when *we target* .. (is this what u're trying to say?)